x/gov/keeper/grpc_query.go

```diff
@@ -91,9 +92,9 @@ func (q queryServer) Proposals(ctx context.Context, req *v1.QueryProposalsReques
 				if err != nil {
 					return nil, err
 				}
-				_, err = q.k.GetDeposit(ctx, p.Id, depositor)
+				has, err := q.k.Deposits.Has(ctx, collections.Join(p.Id, sdk.AccAddress(depositor)))
```

Comment:
behavioural change, perf improvement: uses Has instead of Get in order to avoid getting the value from store and decoding it